velociraptor/autoplotter/lines.py

```diff
@@ -205,20 +208,20 @@ def create_line(
 
         if self.median:
             self.output = lines.binned_median_line(
-                x=masked_x, y=masked_y, x_bins=self.bins
-            )
+                x=masked_x, y=masked_y, x_bins=self.bins,
+                return_additional=True)
         elif self.mean:
             self.output = lines.binned_mean_line(
-                x=masked_x, y=masked_y, x_bins=self.bins
-            )
+                x=masked_x, y=masked_y, x_bins=self.bins,
+                return_additional=True)
         elif self.mass_function:
-            self.output = create_mass_function_given_bins(
+            self.output = *create_mass_function_given_bins(
                 masked_x, self.bins, box_volume=box_volume
-            )
+            ), unyt_array([], units=masked_x.units), unyt_array([], units="dimensionless")
         elif self.histogram:
```

Comment:
For `additional_y` (last element in of the return tuple), I've set `units="dimensionless"` because the `y` array is `None` meaning that I cannot access its attributes (e.g. `y.units`). Perhaps we could improve it by doing something like `units = 1./box_volume.units`. What do you think Josh?

Reply:
Can you not do:

```
elif self.mass_function:
    mass_function_output = create_mass_function_given_bins(
        ...
    )
    self.output = *mass_function_output, unyt_array([], units=mass_function_output[0].units), unyt_array([], units=mass_function_output[1].units)
```

Reply:
The merge somehow occurred before I pushed a commit to change this part of the code (as suggested in your comment). In the merged version, we have

```
elif self.mass_function:
            self.output = *create_mass_function_given_bins(
                masked_x, self.bins, box_volume=box_volume
            ), unyt_array([], units=masked_x.units), unyt_array([], units="dimensionless")
        elif self.histogram:
            self.output = *create_histogram_given_bins(
                masked_x, self.bins, box_volume=box_volume
            ), unyt_array([], units=masked_x.units), unyt_array([], units="dimensionless")
```

Reply:
Yeah, I merged it because I thought we were done :p.

Can you push another commit to this branch?

Reply:
Sure! these changes are now in my branch.

Reply:
Ok, you will need to open a separate pull request.

Reply:
Done. Everything is good now and ready to be merged.